src/Utils/TogglePassword.js

```diff
@@ -30,11 +30,22 @@ class TogglePassword extends Component {
     return (
       <Row>
         <Col xs={10}>
-          <Field label={<FormattedMessage id="ui-organizations.edit.password" />} name={name} id={id} type={this.state.showPassword ? 'text' : 'password'} component={TextField} autoComplete="nope" fullWidth />
+          <Field
+            autoComplete="new-password"
```

Comment:
this solution is descried here https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#Preventing_autofilling_with_autocompletenew-password